src/app/api/v1/fxa-rp-events/route.ts

```diff
@@ -37,16 +37,16 @@ const getJwtPubKey = async () => {
         "Content-Type": "application/json",
       },
     });
-    const { keys } = await response.json();
+    const { keys } = (await response.json()) as { keys: jwkToPem.JWK[] };
     console.info(
       "getJwtPubKey",
       `fetched jwt public keys from: ${jwtKeyUri} - ${keys.length}`
     );
     return keys;
-  } catch (e) {
+  } catch (e: unknown) {
     console.error("getJwtPubKey", `Could not get JWT public key: ${jwtKeyUri}`);
     captureException(
-      new Error(`Could not get JWT public key: ${jwtKeyUri} - ${e}`)
+      new Error(`Could not get JWT public key: ${jwtKeyUri} - ${e as string}`)
```

Comment:
What is `e as string` casting doing?
Is `e` an Error object here? Should this be `${e.message}` or does casting `as string` somehow capture the stack as well?

Reply:
It's a TypeScript type assertion, which only works at build time to appease the TypeScript compiler - in other words, it doesn't change runtime behaviour. The linter is complaining about `e` being of type `any`, i.e. TS is unable to verify that it's actually loggable. Now, I don't know for sure whether it actually casts to a string at runtime, but given that the code already existed, I just assumed that it did.